Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -8,12 +8,13 @@ steps:
    displayName: GitVersion
    inputs:
      preferBundledVersion: false
- - task: NuGetToolInstaller@0
-   displayName: 'Use NuGet 4.3.0'
- - task: NuGetCommand@2
-   displayName: 'NuGet restore ${{ variables.SolutionBaseName }}.sln'
-   inputs:
-     restoreSolution: '${{ variables.SolutionBaseName }}.sln'
+ 
```

Comment:
It's ok to comment a something out whilst your testing but once you're happy with it and confident it's working as expected you should delete the code.  Commented out code makes it hard to read, if anyone wants to see the old code they can go through the commit history